client/api/http_api/http_api.go

```diff
@@ -0,0 +1,46 @@
+package http_api
+
+import (
+	"context"
+	echo_middleware "github.com/labstack/echo/v4/middleware"
+
+	"github.com/labstack/echo/v4"
+	"github.com/lidofinance/dc4bc/client/api/http_api/router"
+	"github.com/lidofinance/dc4bc/client/config"
+	"github.com/lidofinance/dc4bc/client/services/node"
+)
+
+type RESTApiProvider struct {
+	config       *config.HttpApiConfig
+	echoInstance *echo.Echo
+}
+
+func (p *RESTApiProvider) NewServer(config *config.Config, node node.NodeService) error {
+	p.config = config.HttpApiConfig
+
+	p.echoInstance = echo.New()
+
+	p.echoInstance.HideBanner = true
+	p.echoInstance.Debug = false
```

Comment:
It might be a good idea to move Debug to configuration

Reply:
done